packages/api/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -46,21 +47,29 @@ export async function sandboxGetDocRefsAndUpsert({
 
   const patientData = getSandboxSeedData(patient.data.firstName);
   if (!patientData) {
-    await appendDocQueryProgress({
+    const updatedPatient = await appendDocQueryProgress({
       patient: { id: patient.id, cxId: patient.cxId },
       downloadProgress: {
         status: "completed",
       },
       reset: true,
       requestId,
+      source: MedicalDataSource.COMMONWELL,
     });
-    processPatientDocumentRequest(
-      organization.cxId,
-      patient.id,
-      "medical.document-download",
-      MAPIWebhookStatus.completed,
-      []
-    );
+
+    const downloadProgressIsCompleted =
+      updatedPatient.data.documentQueryProgress?.download?.status === "completed";
+
+    if (downloadProgressIsCompleted) {
+      processPatientDocumentRequest(
+        organization.cxId,
+        patient.id,
+        "medical.document-download",
+        MAPIWebhookStatus.completed,
+        []
+      );
+    }
```

Comment:
Do we need to update anything sandbox related for CQ?

Reply:
Im mean technically no - really the cx is going to be receiving dummy data its just going to be through the CW flow we have set up. Ideally we would just move this before the HIE's so its not living on either one